src/Nethermind/Nethermind.Crypto/BlsSigner.cs

```diff
@@ -0,0 +1,86 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+
+namespace Nethermind.Crypto;
+
+using G1 = Bls.P1;
+using G2 = Bls.P2;
+using GT = Bls.PT;
+
+public class BlsSigner
+{
+    internal static readonly string Cryptosuite = "BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_";
```

Comment:
Make this already UTF8 encoded by:
```suggestion
internal static readonly ReadOnlySpan<byte> Cryptosuite = "BLS_SIG_BLS12381G2_XMD:SHA-256_SSWU_RO_POP_"u8;
```